net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -78,6 +78,10 @@ function quagga_ospfd_install_conf() {
 	// generate ospfd.conf based on the assistant
 	if (is_array($config['installedpackages']['quaggaospfd']['config'])) {
 		$ospfd_conf = &$config['installedpackages']['quaggaospfd']['config'][0];
+	} elseif (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospfd'])  ||
+		  isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d']) ||
+	 	  isset($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
```

Comment:
Please fix indent. Multiple lines statements should use 4 spaces on last indent level